generator/src/test/java/com/scottlogic/deg/generator/cucumber/engine/utils/CucumberGenerationConfigSource.java

```diff
@@ -51,7 +50,7 @@ public CucumberGenerationConfigSource(CucumberTestState state) {
 
     @Override
     public boolean getValidateProfile() {
-        return false;
+        return true;
```

Comment:
This should be left as false, this configures whether contradiction-checking is enabled, which is off by default.
Until #768 type mandatoriness/mandation is always on.

Reply:
Agreed, sorry, this snuck in accidentally.